tests/testthat/test-tamper.R

```diff
@@ -0,0 +1,82 @@
+
+context("Tampering")
+
+test_that("We can detect if we are in a pipe", {
+
+  skip(message = "not ready yet for a real test")
```

Comment:
I think we'll want to test this feature automatically even if it's costly, I'd like to take a look at this. My approach would be to define "semi-interactive" behavior (using mocking or an internal switch) that allows specifying the sequence of choices (as list of integers) and the expected output (as list of character vectors or functions that test a character vector). We'd also need a substitute for `browser()`.